odl/operator/default_ops.py

```diff
@@ -754,15 +756,17 @@ class ConstantOperator(Operator):
         ``ConstantOperator(y)(x) == y``
     """
 
-    def __init__(self, vector, domain=None):
+    def __init__(self, vector, domain=None, range=None):
         """Initialize a new instance.
 
         Parameters
         ----------
         vector : `LinearSpaceElement`
             The constant space element to be returned.
         domain : `LinearSpace`, optional
-            Domain of the operator. Default : ``vector.space``.
+            Domain of the operator. Default: ``vector.space``
+        range : `LinearSpace`, optional
```

Comment:
Somewhere in this branch I wanted an operator `A : X -> Y`, where `X` and `Y` are different spaces, and where `A(x) = constant`. Typically a constant operator ;-)

Naming should probably be updated. `constant` sounds good to me.